guides/schema/root_types.md

```diff
@@ -26,16 +27,16 @@ The types are `GraphQL::Schema::Object` classes, for example:
 ```ruby
 # app/graphql/types/query_type.rb
 class Types::QueryType < GraphQL::Schema::Object
-  # ...
+  field :all_posts, [PostType], 'Returns all blog posts', null: false
```

Comment:
Small tactical changes to make the root type document a little more useful. 

Personally, I feel like this page doesn't add enough. It might be good to expand it or push the examples into the "Schema Definition" guide.